metadata_backend/api/operators.py

```diff
@@ -869,19 +861,20 @@ async def create_user(self, data: Tuple) -> str:
         """
         user_data: Dict[str, Union[list, str]] = dict()
 
-        eppn = data[0]  # this also can be sub key
+        externalId = data[0]  # this also can be sub key
         name = data[1]
         try:
-            existing_user_id = await self.db_service.exists_eppn_user(eppn, name)
+            existing_user_id = await self.db_service.exists_user_by_externalId(externalId, name)
             if existing_user_id:
-                LOG.info(f"User with identifier: {eppn} exists, no need to create.")
+                LOG.info(f"User with identifier: {externalId} exists, no need to create.")
                 return existing_user_id
             else:
                 user_data["drafts"] = []
                 user_data["folders"] = []
                 user_data["userId"] = user_id = self._generate_user_id()
                 user_data["name"] = name
-                user_data["eppn"] = eppn
+                user_data["externalId"] = externalId
```

Comment:
can't make multiline suggestion, but `externalId` functions and variables should be named `external_id`

Reply:
I can rename the function but not the name of key of the dict, that is camelcase standard in JSON

Reply:
Yeah, that's what I was also thinking, that snake_case for code, and the dict can stay as it is because it is what it is